### ISSUE TITLE
Application Stops working after 4-5 hours of continuous stumbling

### ISSUE THREAD

**Issue by anupkumarmishra** [mozilla/MozStumbler#1684]
Hi ,

I saw that the application stopped working even when everything was fine . Please look into the screenshot.
![stumble](https://cloud.githubusercontent.com/assets/2819529/8691980/18166622-2ae7-11e5-94e0-91619fc119ad.jpg)

**Comment by crankycoder** [comment]
Thanks @anupkumarmishra !

We need to get some more details to follow up on this.

Are you using v1.7.12 of the stumbler?   What device and version of android are you running?  

The stumbler may 'stop' if Android decides to kill the application - which is almost certainly the case here.    

Your screenshot shows 19% battery - which shouldn't cause the stumbler to stop automatically as the defaults are currently set to 15% in the power saving options.

**Comment by anupkumarmishra** [comment]
Yes , I am using the same version of the application . I am using Moto g device and Android V 5.0.

**Comment by crankycoder** [comment]
I have a Moto G myself running stock Android 5.02 - I think you're just seeing the OS kill the application after a while - this is expected behaviour on Android.  Especially on 5.0 on the Moto G where resources are highly constrained.
